### ISSUE TITLE
Backup crash

### ISSUE THREAD

**Issue by MonomoySama** [LNReader/lnreader#720]
### Steps to reproduce

1. Made backup days ago
2. Now tried restoring backup 

### Expected behavior

Backup shouldn't crash. I previously restored backup with that specific backup file multiple times

### Actual behavior

It's a backup of 78 novels. It's always getting stuck at 6/78. Tried reinstalling, mobile restart multiple times. Waited for 30min but Nothing works.. it's still stuck. so maybe it's a app issue

### LNReader version

1.1.16

### Android version

Android 10

### Device

Realme 7 pro

### Other details

Please fix it as fast as you can. I can't start reading until you fix this

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with a source, I should be opening an issue in the [sources repository](https://github.com/lnreader/lnreader-sources/issues/new/choose).
- [X] I have updated the app to version **[1.1.16](https://github.com/lnreader/lnreader/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by Palloxin** [comment]
If you hope to get it fixed asap send here the Logcat right after the crash (you can trigger it again right after creating the logcat)
If you don't know what it is search on internet, it is not that hard. When you get the file upload it here.

**Comment by rajarshee-harness** [comment]
Could you share the backup file?

**Comment by MonomoySama** [comment]
> Could you share the backup file?

how to share? .json files arent supported to send in reply github says
